R/htmlwidgets.R

```diff
@@ -257,18 +264,22 @@ lookup_widget_html_method <- function(name, package) {
   list(fn = widget_html.default, name = "widget_html.default", legacy = FALSE)
 }
 
-widget_html <- function(name, package, id, style, class, inline = FALSE, ...) {
+widget_html <- function(name, package, id, style, class, inline = FALSE, use_aria = FALSE, ...) {
```

Comment:
Would it make sense for `use_aria` to follow the `htmlwidgets.use_aria` option?

Reply:
I don't think so.  It might make sense to use some test like the `do_use_aria` test, but at this point we can't see the `knitr` options.

Reply:
Well, the knitr options aren't strictly necessary because the global R option is checked before the knitr options. So it'd be reasonable to do either

```suggestion
widget_html <- function(name, package, id, style, class, inline = FALSE, ..., use_aria = getOption("htmlwidgets.USE_ARIA", FALSE)) {
```

or to default to `use_aria = NULL` and then internally call `do_use_aria()`

```r
  args$use_aria <- use_aria %||% do_use_aria()
```

In either case, I'd prefer to add `use_aria` after the `...`, since we're extending the original function signature.

```suggestion
widget_html <- function(name, package, id, style, class, inline = FALSE, ..., use_aria = FALSE) {
```